R/support.R

```diff
@@ -362,17 +362,34 @@ combine_rt <- function(rt, samples) {
 }
 
 
+combine_rt_EpiEstim <- function(rt, samples) {
+  ## Ensure all trajectories are the same length
+  ret <- rt[[1L]]
+  what <- "Rt"
```

Comment:
what is going on here with `what`? you define it, then use the string that you would have defined? Seems like you might have been part way through tidying this up